modules/tinymce/src/plugins/image/main/ts/core/Utils.ts

```diff
@@ -114,12 +113,12 @@ const createImageList = (editor: Editor, callback: (imageList: false | UserListI
   const imageList = Options.getImageList(editor);
 
   if (Type.isString(imageList)) {
-    XHR.send({
-      url: imageList,
-      success: (text) => {
-        callback(JSON.parse(text));
-      }
-    });
+    window.fetch(imageList)
```

Comment:
Instead of doing this can we please add `fetch` to the list of globals we can use in the `.eslintrc` configuration? See https://github.com/tinymce/tinymce/blob/develop/modules/tinymce/.eslintrc.json#L10

Reply:
Done and changed the uses of `window.fetch` to `fetch`